git-publish

```diff
@@ -950,7 +957,7 @@ branch.%s.pushRemote is set appropriately?  (Override with --no-check-url)''' %
 
         git_save_email_lists(topic, to, cc, options.override_cc)
 
-        if not options.pull_request:
+        if not options.pull_request and not options.dry_run:
             # Publishing is done, stablize the tag now
```

Comment:
This patch doesn't cover the `--pull-request` case. No tagging should be performed and nothing should be pushed to the remote.